### PR TITLE
support update brand.

### PR DESCRIPTION
**Pull request by jchen293**
# what
allow users to update the brand through API.

# why
Java library is currently missing this feature.

# how
Add Brand class and the updateBrand method in the User class.

# testing
E2E local test

### CODE REVIEW

**Review by nwithan8 — DISMISSED** [review]
LGTM

File: src/main/java/com/easypost/model/Brand.java [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+
+public class Brand extends EasyPostResource {
+    private String backgroundColor;
+    private String color;
+    private String logo;
+    private String logoHref;
+    private String ad;
+    private String adHref;
+    private String name;
+    private User userID;
```
Comment (Justintime50) [inline src/main/java/com/easypost/model/Brand.java:13]: This is a field and is a `String`

File: src/main/java/com/easypost/model/Brand.java [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+     * Set the ad href of the Brand.
+     *
+     * @param adHref the ad href of the Brand.
+     */
+    public void setAdHref(String adHref) {
+        this.adHref = adHref;
+    }
+
+    /**
+     * Get the name of the Brand.
```
Comment (Justintime50) [inline src/main/java/com/easypost/model/Brand.java:125]:
```suggestion
     * Get the name of the User.
```

File: src/main/java/com/easypost/model/Brand.java [hunk 3]
```diff
@@ -0,0 +1,177 @@
...
+     *
+     * @return the name of the Brand.
```
Comment (Justintime50) [inline src/main/java/com/easypost/model/Brand.java:127]:
```suggestion
     * @return the name of the User.
```

File: src/main/java/com/easypost/model/Brand.java [hunk 4]
```diff
@@ -0,0 +1,177 @@
...
+     */
+    public String getName() {
+        return this.name;
+    }
+
+    /**
+     * Set the name of the Brand.
```
Comment (Justintime50) [inline src/main/java/com/easypost/model/Brand.java:134]:
```suggestion
     * Set the name of the User.
```

File: src/main/java/com/easypost/model/Brand.java [hunk 5]
```diff
@@ -0,0 +1,177 @@
...
+     *
+     * @param name the name of the Brand.
```
Comment (Justintime50) [inline src/main/java/com/easypost/model/Brand.java:136]:
```suggestion
     * @param name the name of the User.
```

**Review by Justintime50 — CHANGES_REQUESTED** [review]

File: src/main/java/com/easypost/model/Brand.java (hunk 1 above)
Comment (jchen293) [reply src/main/java/com/easypost/model/Brand.java:13]: in our API doc, its User object: https://www.easypost.com/docs/api/node#brand

Comment (Justintime50) [reply src/main/java/com/easypost/model/Brand.java:13]: Although the user ID is the id of a `User` object, this is a field that expects/returns a string. Please update it. It should be `string` on our docs as well.

**Review by Justintime50 — APPROVED** [review]

**Review by nwithan8 — APPROVED** [review]

**Pull request merged**